### PR TITLE
Update government-contact-center-council.md

### PR DESCRIPTION
**Pull request by Schultea**
This PR implements the following **changes:**

* CoP Page Update

**URL / Link to page**

https://digital.gov/communities/government-contact-center-council/

### CODE REVIEW

**Comment by saracope** [comment]
@Schultea This has conflicts that need resolved before it can be merged

**Comment by Schultea** [comment]
Updated preview: https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/AS_G3C-Page-Update/communities/government-contact-center-council/

File: content/communities/government-contact-center-council.md [hunk 1]
```diff
@@ -1,53 +1,34 @@
...
+G3C community members join other contact center leaders through our monthly meetings to learn about other agencies’ best practices in contact center operations. The community also hosts webinars, conference calls, v and other networking events. As a member, you have access to other subject matter experts through our LISTSERV where members share ideas, ask questions, and request assistance on their agency’s projects. Our educational forums and discussions help members find innovative methods to improve the customer experience with their agency’s multi-channel contact centers. Membership is open to all federal, tribal, local, and state government employees.
 
 ## Who We Are
 
-The membership consists of more than 275-member contact center managers and directors representing over 60 federal, local and state government agencies and sub-agencies.
+The Government Contact Center Council is an interagency group of contact center directors and managers from across government agencies that work together to improve the performance and efficiency of government contact centers. The Council’s membership includes over 350 members from 60 federal, state, and local agencies.
```
Comment (saracope) [inline content/communities/government-contact-center-council.md:29]: I think the members number reference here should be removed

File: content/communities/government-contact-center-council.md [hunk 2]
```diff
@@ -1,53 +1,34 @@
 ---
-# View this page at https://digital.gov/communities/government-contact-center-council
-# Learn how to edit our pages at https://workflow.digital.gov
-
 slug: government-contact-center-council
 date: 2013-11-21 3:28:02 -0400
```
Comment (saracope) [inline content/communities/government-contact-center-council.md:3]: This should have the CoP lead info added

**Review by saracope — CHANGES_REQUESTED** [review]
Requested a few updates in the comments

File: content/communities/government-contact-center-council.md (hunk 2 above)
Comment (Schultea) [reply content/communities/government-contact-center-council.md:3]: PR for adding Tim as an author: https://github.com/GSA/digitalgov.gov/pull/3619

**Review by ToniBonittoGSA — DISMISSED** [review]

File: content/communities/government-contact-center-council.md (hunk 1 above)
Comment (ToniBonittoGSA) [reply content/communities/government-contact-center-council.md]: done

**Comment by ToniBonittoGSA** [comment]
@Schultea  Couldn't figure out what caused the CircleCI Test to fail; updates are working here https://github.com/GSA/digitalgov.gov/pull/3640

**Pull request closed without merging**